internal_transfer_with_agreed_amount/tests/test_internal_transfer_with_agreed_amount.py

```diff
@@ -2,81 +2,107 @@
 
 
 class TestInternalTransferWithAgreedAmount(TransactionCase):
-    def setUp(self):
-        super(TestInternalTransferWithAgreedAmount, self).setUp()
-        self.currency_usd = self.env.ref("base.USD")
-        self.currency_eur = self.env.ref("base.EUR")
-        self.bank_journal_usd = self.env['account.journal'].create({
-            'name': 'Bank US', 'type': 'bank', 'code': 'BNK68',
-            'update_posted': True})
-        self.bank_journal_eur = self.env['account.journal'].create({
-            'name': 'Bank EUR', 'type': 'bank', 'code': 'BNK67',
-            'currency_id': self.currency_eur.id, 'update_posted': True})
-        self.payment_method_manual = self.env.ref(
-            "account.account_payment_method_manual_in")
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.currency_usd = cls.env.ref("base.USD")
+        cls.currency_eur = cls.env.ref("base.EUR")
+        cls.bank_journal_usd = cls.env["account.journal"].create(
+            {"name": "Bank US", "type": "bank", "code": "BNK68", "update_posted": True}
```

Comment:
Extra: improve indentation, this could be one key per line

Reply:
Indentation changed.